### ISSUE TITLE
Can Livox HAP use this program to calibrate the external IMU?

### ISSUE THREAD

**Issue by hongyiWeng** [hku-mars/LiDAR_IMU_Init#55]
Recently, I purchased a Livox HAP lidar. It does not have a built-in IMU, if I buy an external IMU, can I use this program for calibration?
Thank you.

**Comment by zfc-zfc** [comment]
> Recently, I purchased a Livox HAP lidar. It does not have a built-in IMU, if I buy an external IMU, can I use this program for calibration? Thank you.

Yes, you can use livox avia config to calibrate HAP and external IMU.

**Comment by WadeFrank** [comment]
> Recently, I purchased a Livox HAP lidar. It does not have a built-in IMU, if I buy an external IMU, can I use this program for calibration? Thank you.

你好，请问你使用HAP成功完成标定了吗？
